MC/Config.C

```diff
@@ -24,6 +24,7 @@ static Int_t triggerConfig   = 0.;        // trigger
 static Int_t   pdgConfig       = 0;         // PDG value 
 static Float_t bminConfig      = 0.;        // impact parameter min
 static Float_t bmaxConfig      = 20.;       // impact parameter max
+static Float_t ptHardMinHijing = 3.;        // min cut on ptHard for Hijing simulations
```

Comment:
move to default 2.3 ?

Reply:
Ciao Michael,

I thought that since 2.3 is proven to be not ok, maybe for now we can set the default to 3. I think we should decide on the number when we have the configuration finalized... 

But if you prefer, I put this to 2.3, to be compatible to current value. I will do it.

Chiara

Reply:
Hi Chiara,

as I wrote in my comment, I think for both there is justification. Actually, thinking about it again, I would keep the 3 as default. We just need to remember if we ever do Run-1 productions again ;) 

Cheers,
Michael